avl/lru.node.go

```diff
@@ -29,6 +28,12 @@ import (
 	"sync"
 )
 
+//nolint:lll
```

Comment:
whats that?

Reply:
I'm not 100% sure why the nolint tag was added - it might be because I'm using another genny installation (can switch to another one to double-check).